backend/api/src/model/event.rs

```diff
@@ -53,4 +56,26 @@ impl Event {
 
         Ok(result)
     }
+
+    pub(crate) async fn load_for_series(series_key: Key, context: &Context) -> FieldResult<Vec<Self>> {
+        let result = context.db.get()
+            .await?
+            .query_raw(
+                "select id, title, video, description
+                        from events
+                        where series = $1",
```

Comment:
Typically the line continuations are just indented 4 more spaces, not aligned with anything from the previous line.